two-phase-commit/Makefile

```diff
@@ -0,0 +1,9 @@
+all:
```

Comment:
Again, not ***necessary***, but it would be nice to split compilation and testing into two rules, both of which are called by `all`. Also, some of the other Makefiles run `clean` at the beginning of `all` which ensures a fresh build, would it be preferable to do that here as well?

Reply:
yes, otherwise it won't create the bin directories. Need to fix this.

Reply:
for your first comment, I kind of like running the tests after compilation